docs/software_architecture.md

```diff
@@ -4,15 +4,27 @@ adhocracy+ data model
 adhocracy+ is our reference platform based on our software
 [adhocracy4](https://github.com/liqd/adhocracy4).
 
-## Data Model - Adhocracy4
-The data model of Adhocracy4 can be found in the docs
-[here](https://github.com/liqd/adhocracy4/blob/master/docs/datamodel.md).
+The detailed description of the data model of adhocracy4 can be found
+[in the docs](https://github.com/liqd/adhocracy4/blob/master/docs/datamodel.md).
+Shortly, there are organisations which can create projects. Projects are made up
+of modules, which define which phases are used. The phases determine what the
+users are allowed to do.
 
-## Modules and Phases - Adhocracy4
-The definition and descriptions of modules and phases used in Adhocracy4
-can be found in the docs
+The definition and descriptions of modules and phases can be found
 [here](https://github.com/liqd/adhocracy4/blob/master/docs/phases_and_modules.md).
 
+## Module example in adhocracy+
+An example of a module would be the agenda setting. The description and phases
+used are defined
+[here in the blueprints](./blob/92849ba0d8eb5a27c1571f319be0356db8e07347/apps/dashboard/blueprints.py#L40).
```

Comment:
The links dont work. But is that maybe because its not yet linking from master?

Reply:
Oh, yeah, I might have deleted too much of the link. Will fix that!

Reply:
I only managed to do the links with absolute ones. :woman_shrugging: